products/cdk/get-started.adoc

```diff
@@ -59,25 +59,25 @@ First, download and install a virtualization platform:
 * Windows Hyper-V (For Microsoft Windows)
 
 [.callout-light]
-*NOTE:* If you are using Microsoft Windows, Apple Mac OS X, or a Linux distribution other than Red Hat, Fedora or CentOS, VirtualBox is recommended.
+*NOTE:* If you are using Microsoft Windows, Apple Mac OS X, or a Linux distribution other than Red Hat Enterprise Linux, Fedora, or CentOS, VirtualBox is recommended.
 
 [.callout-light]
-*NOTE:* Hyper-V support is only available on certain versions of Windows such as Windows 10 Pro. To enable Hyper-V right click on the _Start Menu_, select _Programs and Features, click on _Turn Windows features on or off_, make sure the checkbox next to _Hyper-V_ is selected, and click _OK_. Windows may need to restart to install Hyper-V - follow any prompts.
+*NOTE:* Hyper-V support is only available on certain versions of Windows, such as Windows 10 Pro. To enable Hyper-V, right click on the _Start Menu_, select _Programs and Features, click on _Turn Windows features on or off_, make sure the checkbox next to _Hyper-V_ is selected, and click _OK_. Windows may need to restart to install Hyper-V - follow any prompts.
 
 ### Install Vagrant
 
 Next, install Vagrant. All of the Container Development Environment configuration and managment on your development system will be handled for you by Vagrant.
 
 [.callout-light]
-*NOTE:* We recommend you use Vagrant version 1.7.4, or a version *later than 1.8.1* on Microsoft Windows and Apple Mac OS X. Vagrant releases 1.8.0 and 1.8.1 should be avoided due to some bugs that impact proper operation of the kit.
+*NOTE:* Use Vagrant version 1.7.4 with VirtualBox on Microsoft Windows. Use version 1.8.1 and higher with Hyper-V. On Red Hat Enterprise Linux, use Vagrant 1.7.4.
```

Comment:
BTW @LalatenduMohanty hferentschik@ I had problems with latest Windows 10 updates and vagraqnt 1.7.4 (and I had to update virtual box). I'm running Windows Insider (so I get previews) but I suspect we need to be on top of this for when they update windows...